src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs

```diff
@@ -81,6 +82,7 @@ public class EthereumPayoutHandler : PayoutHandlerBase, IPayoutHandler
         private const string TooManyTransactions = "There are too many transactions in the queue";
         private const decimal MaxPayout = 0.1m; // ethereum
         private const double MaxBlockReward = 0.1d; //ethereum
+        private static ConcurrentDictionary<string, string> transactionHashes = new();
```

Comment:
I believe we will run master in a single node as of today. If we plan to run master in multiple nodes then this data should come from DB.

Reply:
yea.  splitting up the master responsibilities to multiple nodes is not easy given the current design.
even if we did, i suspect payouts would all be done by a single node,  balances by another, stats etc....  would be tricky to have multiple nodes doing payouts.